editor/src/components/canvas/remix/remix-error-handling.test-utils.tsx

```diff
@@ -0,0 +1,128 @@
+require('jest-fetch-mock').enableMocks()
+
+import { createModifiedProject } from '../../../sample-projects/sample-project-utils.test-utils'
+import { notLoggedIn } from '../../editor/action-types'
+import { runDOMWalker, switchEditorMode } from '../../editor/actions/action-creators'
+import { EditorModes } from '../../editor/editor-modes'
+import { StoryboardFilePath } from '../../editor/store/editor-state'
+import type { PersistentModel } from '../../editor/store/editor-state'
+import { RegisteredCanvasStrategies } from '../canvas-strategies/canvas-strategies'
+import { mouseClickAtPoint } from '../event-helpers.test-utils'
+import { DefaultStartingFeatureSwitches, renderTestEditorWithModel } from '../ui-jsx.test-utils'
+import type { EditorRenderResult } from '../ui-jsx.test-utils'
+
+async function renderRemixProject(project: PersistentModel) {
+  const renderResult = await renderTestEditorWithModel(
+    project,
+    'await-first-dom-report',
+    DefaultStartingFeatureSwitches,
+    undefined,
+    RegisteredCanvasStrategies,
+    notLoggedIn,
+    false,
+  )
+  await renderResult.dispatch([runDOMWalker()], true)
+  return renderResult
+}
+
+async function clickLinkWithTestId(editor: EditorRenderResult, testId: string) {
+  const targetElement = editor.renderedDOM.queryAllByTestId(testId)[0]
```

Comment:
This should give a sensible error if it can't find it or there's more than one.
```suggestion
  const targetElement = editor.renderedDOM.getByTestId(testId)
```

Reply:
I'm not sure why I used `queryAllByTestId` for this one...